frontend/src/modules/Environments/components/EnvironmentTeams.js

```diff
@@ -368,12 +400,49 @@ export const EnvironmentTeams = ({ environment }) => {
     }
   };
 
-  const handlePageChangeRoles = async (event, value) => {
-    if (value <= roles.pages && value !== roles.page) {
-      await setFilterRoles({ ...filterRoles, page: value });
+  const handlePageChangeRoles = async (page) => {
+    page += 1; //expecting 1-indexing
+    if (page <= roles.pages && page !== roles.page) {
+      await setFilterRoles({ ...filterRoles, page: page });
     }
   };
 
+  const [rowModesModel, setRowModesModel] = React.useState({});
```

Comment:
Can we use `useState()` that is already imported rather than `React.useState()`?